ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -369,10 +375,51 @@ export default function BuildQuote({
             defaultToAll
           />
         </div>
-        {selectedToToken?.address &&
-          selectedToToken?.address !== ETH_SWAPS_TOKEN_OBJECT.address && (
+        {toTokenIsNotEth &&
+          (selectedToToken.occurances === 1 ? (
+            <ActionableMessage
+              message={
+                <div className="build-quote__token-verification-warning-message">
+                  <div className="build-quote__bold">
+                    {t('swapTokenVerificationOnlyOneSource')}
+                  </div>
+                  <div>
+                    {t('verifyThisTokenOn', [
+                      <a
+                        className="build-quote__token-etherscan-link build-quote__underline"
+                        key="build-quote-etherscan-link"
+                        href={`https://etherscan.io/token/${selectedToToken.address}`}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        {t('etherscan')}
+                      </a>,
+                    ])}
+                  </div>
+                </div>
+              }
+              primaryAction={
+                verificationClicked
+                  ? null
+                  : {
+                      label: t('continue'),
+                      onClick: () => setVerificationClicked(true),
+                    }
+              }
+              className="actionable-message--warning actionable-message--with-right-button"
```

Comment:
.... why is there an `actionable-message` class being applied here, in the `build-quote` component? This component should only assign its own classes.

Reply:
Addressed with 270406f29